evap/static/ts/src/quick-review-slider.ts

```diff
@@ -0,0 +1,410 @@
+            foundNext ||= isNext;
+        });
+        if (!foundNext) {
+            this.skipEvaluationButton?.classList.add("d-none");
+        }
+    };
+
+    private updateAnswerIdInputs = () => {
+        let newAnswerId;
+        if (!this.isShowingEndslide()) {
+            assertDefined(this.selectedSlide.dataset.id);
+            newAnswerId = this.selectedSlide.dataset.id;
+        }
+        for (const input of this.slider.querySelectorAll<HTMLInputElement>("input[name=answer_id]")) {
+            input.disabled = input.value !== newAnswerId;
+        }
+    };
+    private updateNavigationButtons = () => {
+        const leftSlides = this.answerSlides.slice(0, this.selectedSlideIndex);
+        const rightSlides = this.answerSlides.slice(this.selectedSlideIndex + 1);
+        const leftReviewed = leftSlides.filter(slide => slide.matches("[data-review]")).length;
+        const rightReviewed = rightSlides.filter(slide => slide.matches("[data-review]")).length;
+
+        this.navigationButtons.left.button.classList.toggle("visible", this.selectedSlideIndex > 0);
+        this.navigationButtons.right.button.classList.toggle("visible", !this.isShowingEndslide());
+        this.navigationButtons.left.reviewedCounter.innerText = leftReviewed.toString();
+        this.navigationButtons.left.unreviewedCounter.innerText = (leftSlides.length - leftReviewed).toString();
+        this.navigationButtons.right.reviewedCounter.innerText = rightReviewed.toString();
+        this.navigationButtons.right.unreviewedCounter.innerText = Math.max(
+            rightSlides.length - rightReviewed,
+            0,
+        ).toString();
+    };
+    private updateAlertSlide = () => {
+        const slidesExist = this.answerSlides.length > 0;
+        const allAreReviewed = this.answerSlides.every(slide => "review" in slide.dataset);
+
+        this.alertSlide.classList.toggle("alert-secondary", !slidesExist);
+        this.alertSlide.classList.toggle("alert-warning", !allAreReviewed);
+        this.alertSlide.classList.toggle("alert-success", slidesExist && allAreReviewed);
+        this.alertContentSpans.unreviewed.classList.toggle("d-none", allAreReviewed);
+        this.alertContentSpans.reviewed.classList.toggle("d-none", !allAreReviewed);
+        this.startOverTriggers.undecided.classList.toggle("d-none", allAreReviewed);
+        this.startOverTriggers.all.classList.toggle("d-none", !slidesExist);
+    };
+
+    //
+    // Sliding
+    //
+    public startOver = (where: StartOverWhere) => {
+        const decided = this.slider.querySelectorAll<HTMLElement>(`[data-layer="${Layer.Answer}"][data-review]`);
+        const undecided = this.slider.querySelectorAll<HTMLElement>(
+            `[data-layer="${Layer.Answer}"]:not([data-review])`,
+        );
+        this.answerSlides = Array.from(decided).concat(Array.from(undecided));
+
+        const startIndex = where === StartOverWhere.Undecided && undecided.length > 0 ? decided.length : 0;
```

Comment:
I'd prefer some parentheses here (I don't have an operator precedence table in my head).

Reply:
prettier says "but you should" :/ I moved the condition into another variable